src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -1,47 +1,73 @@
 #include <gtest/gtest.h>
 
-#include "software/ai/ai_wrapper.h"
 #include "software/backend/simulator_backend.h"
 #include "software/gui/visualizer_wrapper.h"
-#include "software/typedefs.h"
+#include "software/test_util/test_util.h"
 #include "software/util/logger/init.h"
+#include "software/util/time/duration.h"
+
 
 class SimulatedTest : public ::testing::Test
 {
-   public:
-    void enableVisualizer()
-    {
-        // TODO: how to do this without argc and argv? (#768)
-
-        //        visualizer = std::make_shared<VisualizerWrapper>(argc, argv);
-        //
-        //        backend->Subject<World>::registerObserver(visualizer);
-        //        ai->Subject<AIDrawFunction>::registerObserver(visualizer);
-        //        ai->Subject<PlayInfo>::registerObserver(visualizer);
-        //        backend->Subject<RobotStatus>::registerObserver(visualizer);
-    }
-
    protected:
-    static void SetUpTestSuite()
+    void SetUp() override
     {
         Util::Logger::LoggerSingleton::initializeLogger();
+        backend = std::make_shared<SimulatorBackend>(
+            Duration::fromMilliseconds(5), Duration::fromSeconds(1.0 / 30.0),
+            SimulatorBackend::SimulationSpeed::FAST_SIMULATION);
     }
 
-    void SetUp() override
+    /**
+     * This function enables the Visualizer while a test is running, so that the test can
+     * be debugged Visually. Simply call this function at the start of the test(s) you
+     * want to show in the Visualizer. Make sure to run the test targets with 'bazel run'
+     * instead of 'bazel test' or the Visualizer won't work.
+     */
```

Comment:
We should define this class like we do normally, with a `.h` and `.cpp` file, and the declarations in the `.h`

Reply:
This is a gtest, and we currently don't split gtest fixtures into `.h` and `.cpp` files. I'm not sure it's necessary in this case?